src/NHSD.BuyingCatalogue.Identity.Api/Extensions/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Http;
+using NHSD.BuyingCatalogue.Identity.Common.Constants;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Extensions
+{
+    public static class HttpRequestExtensions
+    {
+        public static bool ShowCookieConsent(this HttpRequest httpRequest) =>
+            httpRequest != null && !httpRequest.Cookies.ContainsKey(Cookies.BuyingCatalogueConsent);
```

Comment:
`!= null` should be `is not null` to be consistent with the style used elsewhere in the solution.

Reply:
corrected